docs/docsource/operation/platform_operations/retrieve_access_tokens.rst

```diff
@@ -6,46 +6,51 @@ Retrieve Access Tokens
 Prerequisites
 ~~~~~~~~~~~~~
 
-- jq and curl installed
-- Access from the browser user machine
+-kubectl, jq, and curl installed
 
 Procedure
 ~~~~~~~~~
 
-#. Retrieve ``ACCESS_TOKEN``, ``TOKEN``, and ``INSTANCE_ID``:
+.. note::
 
-   .. code::
-      
-      OIDC_REALM='zenko-realm'
+   In the following commands, replace ``<username>`` and ``<password>`` with
+   the user credentials provisioned for the OIDC service, and ``<zenko-name>``
+   with the name of the deployed Zenko instance.
+
+.. note::
+
+   In the following example, ``Keycloak`` is used as the OIDC provider, and
+   ``https://ui.zenko.workloadplane.scality.local`` is the url for making
+   the request to the ``Keycloak`` service. In addition, ``artesca`` and
+   ``zenko-ui`` are the example Keycloak realm and client-id values,
+   respectively.
+
+#. Declare variables 
+
+   .. code::  
+
+      OIDC_REALM='artesca'
       OIDC_CLIENT_ID='zenko-ui'
-      OIDC_USER='zenko-tester'
-      OIDC_USER_PASSWORD='password'
-      ZENKO_NAME='zenko-instance'
-      
-      ACCESS_TOKEN=$(
-          curl -s -k "http://keycloak.zenko.local/auth/realms/${OIDC_REALM}/protocol/openid-connect/token" \
-              -d 'scope=openid' \
-              -d "client_id=${OIDC_CLIENT_ID}" \
-              -d "username=${OIDC_USER}" \
-              -d "password=${OIDC_USER_PASSWORD}" \
-              -d "grant_type=password" | \
-              jq -cr '.access_token'
-      )
-      
+      OIDC_USER='<username>'
+      OIDC_USER_PASSWORD='<password>'
+
+#. Retrieve ``TOKEN``:
+
+   .. code::
+
       TOKEN=$(
-          curl -s -k "http://keycloak.zenko.local/auth/realms/${OIDC_REALM}/protocol/openid-connect/token" \
+          curl -s -k "http://ui.zenko.workloadplane.scality.local/auth/realms/${OIDC_REALM}/protocol/openid-connect/token" \
```

Comment:
```suggestion
          curl -s -k "https://ui.zenko.workloadplane.scality.local/auth/realms/${OIDC_REALM}/protocol/openid-connect/token" \
```
Not working in http since 1.1